scipy/optimize/slsqp.py

```diff
@@ -172,7 +172,7 @@ def fmin_slsqp(func, x0, eqcons=(), f_eqcons=None, ieqcons=(), f_ieqcons=None,
          6 : Singular matrix C in LSQ subproblem
          7 : Rank-deficient equality constraint subproblem HFTI
          8 : Positive directional derivative for linesearch
-         9 : Iteration limit exceeded
+         9 : Iteration limit Reached
```

Comment:
reached does not require capitalisation.

Reply:
I fixed it.

Reply:
`Reached` --> `reached`

Reply:
Oh. sorry. I fixed another one. I fixed this one.